src/pages/Lending/hooks/usePoolDataQuery.ts

```diff
@@ -76,6 +76,7 @@ export const usePoolDataQuery = ({ poolAssetId }: { poolAssetId: string }) => {
         collateralizationRatioPercentDecimal,
         tvl,
         maxSupplyFiat,
+        status: poolInfo.status,
```

Comment:
suggestion: we probably want to use `isTradingActive` here for consistency - that's an extra network call for data we already have, but would make things a lot easier to monkey patch and debug 🙏🏽

Reply:
Fixed in [8adb4bc](https://github.com/shapeshift/web/pull/7191/commits/8adb4bc5d03daab4737af445ebe41feedd9d88de)